src/main/resources/video.properties

```diff
@@ -0,0 +1,6 @@
+video.folder=${user home}/video
```

Comment:
You can store these details in a separate property file, but what's the point in that? this config works only for video for allure. Do you see situations when you might need other configs?
I would recommend to store them in the class with video recorder so that it's all together. Not sure you will ever change them tbh

Reply:
There is no class with a video recorder :) I have a video testng record dependency that relies on these properties, if they are deleted, the video recording won't work